main.go

```diff
@@ -129,6 +130,8 @@ func launch(args args) (func(), error) {
 		}, nil
 
 	case "agent":
+		// metrics.RegisterAgentMetrics()
```

Comment:
What is this about?

Reply:
I doesn't exists yet, IIRC

Reply:
It's there as a placeholder to remind me to do it, at some point.

Reply:
Gotcha.